batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -33,12 +33,12 @@
 	
 	void setOpenmrsServerUrl(String value);
 	
-	// TODO merge this with `openmrsServerUrl` to `fhirBaseUrl`
+	// TODO merge this with `openmrsServerUrl` after deprecating the atom-feed mode.
 	@Description("OpenMRS server fhir endpoint")
 	@Default.String("/ws/fhir2/R4")
-	String getServerFhirEndpoint();
+	String getOpenmrsfhirBaseEndpoint();
 	
-	void setServerFhirEndpoint(String value);
+	void setOpenmrsfhirBaseEndpoint(String value);
```

Comment:
Sorry forgot the point about capitalizing `Fhir` last night. Please fix it in the streaming mode too and update the docs as well.

```suggestion
	void setOpenmrsFhirBaseEndpoint(String value);
```